doc-content/enterprise-only/springboot/bus-app-create_proc.adoc

```diff
@@ -1,36 +1,52 @@
+
+.Prerequisites
+* Apache Maven 3.5 or higher
 
 .Procedure
-. Enter the following URL in a web browser:
+Enter one of the following commands to create your Spring Boot business application project. In these commands, replace `<project>` with the name of your project:
+
+* To create a business asset project that contains business processes, rules, and forms:
++
+[source,subs="attributes+"]
+----
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-kjar-archetype -DarchetypeVersion={MAVEN_ARTIFACT_VERSION} -DgroupId=com.company -DartifactId=<project>-kjar -Dversion=1.0-SNAPSHOT -Dpackage=com.company
+----
 +
-[source]
+This command creates `org.kie:kie-kjar-archetype:{MAVEN_ARTIFACT_VERSION}`.
+
+* To create a data model asset project that provides common data structures that are shared between the service projects and business assets projects:
++
+[source,subs="attributes+"]
 ----
-https://start.jbpm.org
+mvn archetype:generate -B -DarchetypeGroupId=org.kie -DarchetypeArtifactId=kie-model-archetype -DarchetypeVersion={MAVEN_ARTIFACT_VERSION} -DgroupId=com.company -DartifactId=<project>-model -Dversion=1.0-SNAPSHOT -Dpackage=com.company.model
 ----
++
+This command creates `org.kie:kie-kjar-archetype:{MAVEN_ARTIFACT_VERSION}`.
 
-. Click *Configure your business application*.
-ifdef::PAM[]
-. Click *Business Automation* and click *Next*.
-endif::[]
-ifdef::DM[]
-. Click *Decision Management* and click *Next*.
-endif::[]
-. Enter a package and application name.
-. Select *Enterprise {PRODUCT_VERSION}* from the *Version* menu and click *Next*.
-+
-[NOTE]
-====
-You must select *Enterprise {PRODUCT_VERSION}* to create a {PRODUCT} business application.
-====
-. Select the project types that you want to include in your project. You can include more than one project type.
-+
-* *Business Assets*: Contains business processes, rules, and forms and are easily imported into {CENTRAL}.  Select *Dynamic Assets* instead if you want to add adaptive and dynamic assets such as cases.
-* *Data Model*: Provides common data structures that are shared between the service projects and business assets projects. This enables proper encapsulation, promotes reuse, and reduces shortcuts. Each service project can expose its own public data model.
-* *Service*: Includes business logic that operates your business.
-. Click *Generate business application*.
-+
-The `<BUSINESS-APPLICATION>.zip` file downloads, where `<BUSINESS-APPLICATION>` is the name that you entered in the *Application Name* box.
-. Extract the `<BUSINESS-APPLICATION>.zip` file.
+* To create a service project, a deployable project that provides a service with various capabilities including the business logic that operates your business, enter one of the following commands:
+** To create a business application service project (the default configuration) that includes features for process management, case management, decision management, and optimisation:
```

Comment:
Not sure if optimization (with z) is more correct or they are interchangeable.

Reply:
Good catch, @gmunozfe. We use American English so should have the z.